virttest/qemu_storage.py

```diff
@@ -532,7 +532,8 @@ def _parse_options(self, params):
             "preallocated": (None, "preallocation", ("qcow2", "raw", "luks")),
             "image_cluster_size": (None, "cluster_size", ("qcow2",)),
             "lazy_refcounts": (None, "lazy_refcounts", ("qcow2",)),
-            "qcow2_compatible": (None, "compat", ("qcow2",))
+            "qcow2_compatible": (None, "compat", ("qcow2",)),
+            "image_extent_size_hint": (None, "extent_size_hint", ("qcow2", "raw", "luks"))
```

Comment:
we may need to give a brief instruction to image_extent_size_hint in base.cfg, is it for qemu-img cmdline only and if it is also supported in qemu-kvm cmdline?

Reply:
@zhencliu, qemu-kvm cmdline don't support it, only qemu-img. And I have added the notes in base.cfg, you can help check it, thanks.